api/client/build.go

```diff
@@ -131,11 +130,19 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		return fmt.Errorf("cannot canonicalize dockerfile path %s: %v", relDockerfile, err)
 	}
 
-	var includes = []string{"."}
+	var (
```

Comment:
I personally hate `var` blocks :) variables should be defined where it need, includes can be not needed at all later because of error.
I noticed this because of block, but I think it's good time to move includes below to its usage